ezyrb/cvt.py

```diff
@@ -90,10 +95,56 @@ def _compute_simplex_volume(simplex_vertices):
 
 		return volume
 
-	def compute_errors(self):
+	def set_error_estimator(self, func):
+		"""
+		Allows to user to specify a custom function to estimate error in
+		loo compute error procedure.
+
+		:param function func: the function to use as error estimator: it have
+			to take as input a 1D numpy array and return a float
+		"""
+		self.__error_estimator_func = func
```

Comment:
http://stackoverflow.com/questions/1301346/the-meaning-of-a-single-and-a-double-underscore-before-an-object-name-in-python